specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json

```diff
@@ -0,0 +1,1725 @@
+          "description": "The properties of a configuration."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Represents a Configuration."
+    },
+    "ConfigurationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Configuration"
+          },
+          "description": "The list of server configurations."
+        }
+      },
+      "description": "A list of server configurations."
+    },
+    "OperationDisplay": {
+      "properties": {
+        "provider": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Operation resource provider name."
+        },
+        "resource": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource on which the operation is performed."
+        },
+        "operation": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Localized friendly name for the operation."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Operation description."
+        }
+      },
+      "description": "Display metadata associated with the operation."
+    },
+    "Operation": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the operation being performed on this particular object."
+        },
+        "display": {
+          "$ref": "#/definitions/OperationDisplay",
+          "readOnly": true,
+          "description": "The localized display information for this particular operation or action."
+        },
+        "origin": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The intended executor of the operation.",
+          "enum": [
+            "NotSpecified",
+            "user",
+            "system"
+          ],
+          "x-ms-enum": {
+            "name": "OperationOrigin",
+            "modelAsString": true
+          }
+        },
+        "properties": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          },
+          "readOnly": true,
+          "x-ms-client-flatten": false,
+          "description": "Additional descriptions for the operation."
+        }
+      },
+      "description": "REST API operation definition."
+    },
+    "OperationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "description": "The list of resource provider operations."
+        }
+      },
+      "description": "A list of resource provider operations."
+    },
+    "LogFileProperties": {
+      "properties": {
+        "name": {
```

Comment:
name is repeated so many times. Its in proxy resource model, then another top level property called name, and then again here. Please keep just the one under the Proxy rsource reference.

Reply:
Will update the API and then swagger post GA.